PMSCore.Test/TesterTest.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace PMSCore.Test
+{
+    public class TesterTest
+    {
+        [Fact]
+        public void Constructor_InitializesATesterWithDefaultParameters()
+        {
+            // Arrange + Act
+            var tester = new Tester("Tester1");
+
+            // Assert
+            Assert.Equal("Tester1", tester.UserName);
+            Assert.Equal(Permission.TESTER, tester.GetPermission());
+            Assert.NotNull(tester); // To ensure an object is created
+        }
+
+        [Fact]
+        public void AddTestCase_AddsAValidTestCaseToList()
+        {
+            // Arrange
+            var tester = new Tester("Tester1");
+
+            // Act
+            tester.AddTestCase("Test case 1");
+
+            // Assert
+            Assert.Contains("Test case 1", tester.TestCases);
+        }
+
+        [Fact]
+        public void AddTestCase_ShowsErrorForInvalidTestCase()
+        {
+            // Arrange
+            var tester = new Tester("Tester1");
+
+            // Act
+            tester.AddTestCase("");
+
+            // Assert
+            Assert.Empty(tester.TestCases);
+        }
+
+        [Fact]
+        public void ViewTestCases_ShouldShowsAllTestCases()
+        {
+            // Arrange
+            var tester = new Tester("Tester1");
+            tester.AddTestCase("Test case 1");
+            tester.AddTestCase("Test case 2");
+
+            // Act
+            var consoleOutput = CaptureConsoleOutput(() => tester.ViewTestCases());
+
+            // Assert
+            Assert.Contains("Test case 1", consoleOutput);
+            Assert.Contains("Test case 2", consoleOutput);
+        }
+
+        [Fact]
+        public void SubmitBugReport_AValidReport_AddsToTheBugReports()
+        {
+            // Arrange
+            var tester = new Tester("Tester1");
+
+            // Act
+            tester.SubmitBugReport("Bug 1", "High");
+
```

Comment:
in this line: tester.SubmitBugReport("Bug 1", "High");

change ViewBugReports to return a formatted string or list to optimize testing.